R/write_secuTrial.R

```diff
@@ -1,11 +1,14 @@
 #' Write secuTrial exports to other formats
-#' @description Convert the export prepared in R and export it SPSS, Stata or SAS using the haven package.
+#' @description Convert the export prepared in R and export it to
+#'              SPSS (sav), Stata (dta) or SAS (sas, xpt version 8)
+#'              using the haven package.
 #' @name write_secuTrial
 #' @param object secuTrialdata object
 #' @param path directory where the files should be saved
-# @param format format in which to save the export (one of "dta", "sas", "sav", "xpt")
-#' @param format format in which to save the export (one of "dta", "sas", "sav")
-#' @details Due to variable naming limitations in other packages, date variables are appended with _d (rather than _date), datetime/POSIX variables are appended with _dt (rather than _datetime) and factors with _f (rather than _factor).
+#' @param format format in which to save the export (one of "dta", "sas", "sav", "xpt")
+#' @details Due to variable naming limitations in other packages, date variables are
+#'          appended with _d (rather than _date), datetime/POSIX variables are appended
```

Comment:
do we need other notes about name truncation?

Reply:
The thing is that I don't really know which rules are applied by haven.

Reply:
sounds fair enough to me

Reply:
Ok, I'll make the adjustments.